react-native-in-app-payments-quickstart/README.md

```diff
@@ -111,3 +111,6 @@ Square provides a sample backend service for this quickstart that can easily be
 [root README]: ../README.md
```

Comment:
After the upgrade, do we need update the steps in this README?

Reply:
No it's not require.